app/contacts/new/page.js

```diff
@@ -0,0 +1,65 @@
+'use client';
+import React, { useState } from 'react';
+import UserContext from '@/app/contexts/UserContext';
+import { useContext } from 'react';
+import { useRouter } from 'next/navigation';
+
+const AddNew = () => {
+
+  const context = useContext(UserContext);
+  const router = useRouter();
+
+  const generateID = () => Math.round(Math.random() * 100000000);
+
+  const [pic, setPic] = useState('https://');
+
+  const handlePicChange = (e) => {
+    if (!e.target.value.startsWith("https://")) {
```

Comment:
not sure why you need to have this logic added in - should just use the image as is regardless if its http or https

Reply:
Thanks for the feedback! This was because next's Image component would throw an error if src is passed something that doesn't start with http or https. If random letters were typed for the URL, this logic at least has next Image try to resolve it first then I can use the onError to handle the error. Maybe there was a better way to do this?